run-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+file=$1
+
+function timeit {
+  bash timeit.sh "$1" "$2 40"  >> runtime.txt
+}
+
+if [[ $file == */js/* ]]; then
+  echo "Testing JavaScript file: $file"
+  timeit "Node" "node $file"
+  timeit "Bun" "bun $file"
+  timeit "Deno" "deno $file"
+elif [[ $file == */php/* ]]; then
+  timeit "PHP" "php $file"
+elif [[ $file == */py/* ]]; then
+  echo "Testing Python file: $file"
+  timeit "PyPy" "pypy $file"
+  timeit "Python" "python3 $file"
+elif [[ $file == */r/* ]]; then
+  timeit "R" "Rscript $file"
+elif [[ $file == */ruby/* ]]; then
+  timeit "Ruby" "ruby $file"
```

Comment:
this feels like a burden on the developers to have to update 2 files in order to add a new language. an ideal solution is for `run.sh` to accept an optional argument where the language to test is specified, where the logic here can kick in.

Reply:
Yes it can be simplified later (in separate PR), once the workflow starts running smoothly.
As of now, merging the 2 seems pre-optimization and may impact existing design.